library/multipicker/src/main/java/im/vector/lib/multipicker/utils/ContentResolverUtil.kt

```diff
@@ -113,7 +116,14 @@ internal fun Uri.toMultiPickerVideoType(context: Context): MultiPickerVideoType?
     }
 }
 
-fun Uri.toMultiPickerAudioType(context: Context): MultiPickerAudioType? {
+private suspend fun ContentResolver.openFileDescriptorOnIo(uri: Uri, mode: String) =
+        withContext(Dispatchers.IO) {
```

Comment:
Not a blocker, It would be better especially for tests, if the value is not hardcoded as  `Dispatchers.IO`, it could be injected or passed from the constructor.